pages/get_involved.md

```diff
@@ -7,57 +7,105 @@ subtitle: Ways to contribute to the US-RSE Association
 
 The US-RSE Association is a new organization that needs community involvement to grow and mature.  Below are some ways members can get more involved in building this organization. Have additional ideas for a working group or committee?  Let us know and bring the discussion to slack.
 
-If you have another idea but aren’t sure how to get started, the Steering Committee would be happy to help. Reach out on Slack or email us all at [contact@us-rse.org](mailto:contact@us-rse.org).
+If you have another idea but aren’t sure how to get started, the Steering Committee would be happy to help. Reach out on Slack (@sc) or email us all at [contact@us-rse.org](mailto:contact@us-rse.org).
+
+<br>
+
+# Easy
 
 ---
 
-## US-RSE Website Committee Charter
+### Invite a New Member
 
-We are seeking help to develop a charter document and determine the initial members of the US-RSE Website Committee. Once ready, the document will be reviewed and ratified by the US-RSE Steering Committee.
+Email at least one other person who might be interested in the US-RSE association and invite them to join (via the link on the main website page: [https://us-rse.org/](https://us-rse.org/)).  
 
-There are two main goals for the committee: 1. Provide guidance for website growth and direction 2. Technical website management
+### Introduce Yourself
 
-As the US-RSE association grows, it is important that we have the proper guidance and support structure in place to ensure the website reflects the values and beliefs of the organization as a whole.
+Have you been lurking on Slack for a while?  Introduce yourself on the #introductions channel, even if you’re not new to the group.
 
-* **Get involved**: join the [#website channel on Slack](https://usrse.slack.com/messages/CJTL0MWUE) 
-* **Contact**: Lance Parsons <[lparsons@princeton.edu](mailto:lparsons@princeton.edu)>.
+### Welcome New Members
 
----
+Join the #introductions channel on Slack and welcome new members.  
+
+### Share a Resource/Ask a Question
+
+Found a good resource that’s worth sharing? Post it on Slack! Looking for a good resource or some advice?  Post it on Slack!  Looking for a little more guidance? See How to Share with the Community
```

Comment:
```suggestion
Whether you've found a good resource that’s worth sharing, or are looking for some advice, you are encouraged to post your ideas and questions on Slack!  Visit https://usrse.slack.com/, or reach out to contact@us-rse.org for an invitation.
```

Reply:
I like the first sentence, but we want people to join via the form, so I'm going to reword to that part.